docs/js/home.js

```diff
@@ -216,3 +233,38 @@ Star.prototype.draw = function() {
   context.fill();
   context.restore();
 };
+
+// http://stackoverflow.com/questions/14012766/detecting-whether-two-divs-overlap
+function overlap($div1, $div2) {
+  var rect1 = $div1[0].getBoundingClientRect();
+  var rect2 = $div2[0].getBoundingClientRect();
+
+  return !(rect1.right < rect2.left ||
+    rect1.left > rect2.right ||
+    rect1.bottom < rect2.top + 80 ||
```

Comment:
Sorry this is a top offset to have it exactly change color on the right stop (middle of the svg path), will refactor this later on